lib/elixir/test/elixir/enum_test.exs

```diff
@@ -47,6 +47,19 @@ defmodule EnumTest do
     assert Enum.at([2, 4, 6], -4) == nil
   end
 
+  test "chunk/3" do
+    enum = Enum
+    enumerable = [1, 2, 3, 4, 5]
+    assert enum.chunk(enumerable, 2, 1) == enum.chunk(enumerable, 2, 1, :discard)
+  end
+
+  test "chunk/4" do
+    enum = Enum
+    enumerable = [1, 2, 3, 4, 5]
+    assert enum.chunk(enumerable, 2, 1, :discard) == [[1, 2], [2, 3], [3, 4], [4, 5]]
```

Comment:
We should remove this line because `:discard` was never a valid argument for `Enum.chunk`.  It is an implementation detail that is "leaking".

Reply:
ok, makes sense